### DISCUSSION TITLE
FullStack - Paginação Blazor

### DISCUSSION THREAD

**Discussion by rianck** [balta-io/forum#621] [Tira Dúvidas] [answered]
Não sei se estou cego , mas não achei o tratamento e implementação da paginação no lado do Blazor....

**Comment by aps180180** [comment, marked as answer]
@andrebaltieri  eu também percebi isso!
Seria possível por gentileza, fazer uma aula ou artigo sobre a paginação no front blazor?
Grato!

**Reply by andrebaltieri** [reply]
Claro meu querido! Vou preparar!

**Reply by rianck** [reply]
top no aguardo
